app/models/api_models.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+from models.coordinate import Coordinate
```

Comment:
`app`から絶対パスで指定しないとuvicorn側がエラー吐いちゃうので対応お願いします
```suggestion
from app.models.coordinate import Coordinate
```

Reply:
これよく見てみたら何にも使ってないやこのimport
修正します感謝！